frame/fast-unstake/src/weights.rs

```diff
@@ -70,29 +72,42 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Balances Locks (r:1 w:1)
 	// Storage: Staking Ledger (r:0 w:1)
 	// Storage: Staking Payee (r:0 w:1)
-	fn on_idle_unstake() -> Weight {
-		// Minimum execution time: 82_426 nanoseconds.
-		Weight::from_ref_time(83_422_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(11 as u64))
-			.saturating_add(T::DbWeight::get().writes(6 as u64))
+	/// The range of component `b` is `[1, 32]`.
+	fn on_idle_unstake(b: u32, ) -> Weight {
+		// Minimum execution time: 103_705 nanoseconds.
+		Weight::from_ref_time(72_740_820)
+			// Standard Error: 47_638
+			.saturating_add(Weight::from_ref_time(37_481_933).saturating_mul(b.into()))
+			.saturating_add(T::DbWeight::get().reads(6))
+			.saturating_add(T::DbWeight::get().reads((6_u64).saturating_mul(b.into())))
+			.saturating_add(T::DbWeight::get().writes(1))
+			.saturating_add(T::DbWeight::get().writes((5_u64).saturating_mul(b.into())))
 	}
 	// Storage: FastUnstake ErasToCheckPerBlock (r:1 w:0)
 	// Storage: Staking ValidatorCount (r:1 w:0)
-	// Storage: ElectionProviderMultiPhase CurrentPhase (r:1 w:0)
 	// Storage: FastUnstake Head (r:1 w:1)
-	// Storage: FastUnstake Queue (r:2 w:1)
 	// Storage: FastUnstake CounterForQueue (r:1 w:1)
+	// Storage: ElectionProviderMultiPhase CurrentPhase (r:1 w:0)
+	// Storage: FastUnstake Queue (r:33 w:32)
```

Comment:
This assumes that we are over-counting here, we are assuming that we always have to fill the `Head` with `Queue`, which is totally not the case.